src/app/lorax/services/data.js

```diff
@@ -20,46 +20,31 @@ define([
         this._mainData, this._requestingMain, this._mainDefer;
         this._mapData, this._requestingMap, this._mapDefer;
 
-        function _buildMainEndpoint() {
-            return [
-                '/data',
-                'base',
-                'main.json'
-            ].join('/');
+        function _buildLocaleMainEndpoint() {
```

Comment:
Because this no longer accepts the `locale` param, I assume there need to be some other changes to the code that calls this function and the infographic one?